python/friesian/democode/train_2tower.py

```diff
@@ -0,0 +1,175 @@
+    elif args.cluster_mode == "standalone":
+        sc = init_orca_context("standalone", master=args.master,
+                               cores=args.executor_cores, num_nodes=args.num_executor,
+                               memory=args.executor_memory,
+                               driver_cores=args.driver_cores, driver_memory=args.driver_memory,
+                               conf=spark_conf,
+                               init_ray_on_spark=True)
+    elif args.cluster_mode == "yarn":
+        sc = init_orca_context("yarn-client", cores=args.executor_cores,
+                               num_nodes=args.num_executor, memory=args.executor_memory,
+                               driver_cores=args.driver_cores, driver_memory=args.driver_memory,
+                               conf=spark_conf, extra_python_lib="../example/two_tower/model.py",
+                               object_store_memory="8g",
+                               init_ray_on_spark=True)
+    elif args.cluster_mode == "spark-submit":
+        sc = init_orca_context("spark-submit")
+    else:
+        invalidInputError(False,
+                          "cluster_mode should be one of 'local', 'yarn', 'standalone' and"
+                          " 'spark-submit', but got " + args.cluster_mode)
+
+    full = FeatureTable.read_parquet(args.data_dir + "/full")
+    stats = full.get_stats(["user", "item", "zipcode", "gender", "age", "occupation", "zipcode", "genres"], "max")
+    for key in stats.keys():
+        stats[key] += 1
+    print(stats)
+    for items in stats:
+        print(items)
+    train_tbl, test_tbl = full.random_split([0.8, 0.2], seed=1)
+    user_info = ColumnInfoTower(indicator_cols=["gender", "age", "occupation"],
+                                indicator_dims=[stats["gender"], stats["age"], stats["occupation"]],
+                                embed_cols=["user", "zipcode"],
+                                embed_in_dims=[stats["user"], stats["zipcode"]],
+                                embed_out_dims=[16, 16],
+                                numerical_cols=["user_visits", "user_mean_rate"],
+                                numerical_dims=[1, 1],
+                                name="user")
+    item_info = ColumnInfoTower(indicator_cols=["genres"],
+                                indicator_dims=[stats["genres"]],
+                                embed_cols=["item"],
+                                embed_in_dims=[stats["item"]],
+                                embed_out_dims=[16],
+                                numerical_cols=["item_visits", "item_mean_rate"],
+                                numerical_dims=[1, 1],
+                                name="item")
+    train_config = {"lr": 1e-3,
+                    "user_col_info": user_info,
+                    "item_col_info": item_info,
+                    "inter_op_parallelism": 4,
+                    "intra_op_parallelism": args.executor_cores}
+
+    train_tbl.df.printSchema()
```

Comment:
do not use spark df in the example; if needed, add the operator to Table or FeatureTable.